datastax/src/main/scala/com/avast/sst/datastax/DatastaxHelper.scala

```diff
@@ -0,0 +1,20 @@
+package com.avast.sst.datastax
+
+import com.datastax.oss.driver.api.core.config.{DriverOption, ProgrammaticDriverConfigLoaderBuilder => DriverBuilder}
+
+import scala.collection.JavaConverters._
+import scala.concurrent.duration.Duration
+
+/** Helper functions to construct Datastax session using Java builder. */
+object DatastaxHelper {

```

Comment:
Please make this helper private to the module, no need to export it for everyone.

Reply:
Done